docs/tutorial.md

```diff
@@ -11,22 +11,33 @@ also helps operate the charm by liaising with the Canonical Observability Stack
 tutorial will walk you through each step of deployment to get a basic WordPress deployment.
 
 ## Requirements
+- A working station, e.g., a laptop, with amd64 architecture.
+- Juju 3 installed and bootstrapped to a MicroK8s controller. You can accomplish this process by using a Multipass VM as outlined in this guide: [Set up / Tear down your test environment](https://juju.is/docs/juju/set-up--tear-down-your-test-environment)
 
-To deploy wordpress-k8s charm, you will need a Juju bootstrapped with any Kubernetes controller.
-To see how to bootstrap your Juju installation with MicroK8s, please refer to the documentation
-on MicroK8s [installation](https://juju.is/docs/olm/microk8s).
+For more information about how to install Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
 
-## Steps
-### Set up the tutorial model
+:warning: When using a Multipass VM, make sure to replace `127.0.0.1` IP addresses with the
```

Comment:
Is there a 127.0.0.1 IP address used in this tutorial? This warning might not be necessary for this tutorial?